uoftscrapers/scrapers/utils/scraper.py

```diff
@@ -41,6 +41,7 @@ def get(url, params=None, cookies=None, headers=None, json=False, max_attempts=1
                     doc = r
                 else:
                     sleep(0.5)
+                    attempts += 1
```

Comment:
Not sure if this was by design, but whenever the request 404'd, it'd go on forever.

Try it with UTSG `Exams` (June 2016)